samples/streamlit-in-snowflake/sis-deploy/README.md

```diff
@@ -30,9 +35,10 @@ schema = "<SCHEMA>"
 warehouse = "<WAREHOUSE>"
 ```
 
-# Set up local conda environment
+# Set up local environment
 
 ```sh
+pip install snowflake-cli-labs==1.2.1
 brew install miniconda`
```

Comment:
Might want to note that these are the instructions for Mac/Linux, and will be different on Windows